PYCS_Language/Constants.cs

```diff
@@ -5,7 +5,6 @@ public class BasicOperationsConstants
         public const string LET = "let";
         public const string IF = "if";
         public const string ELSE = "else";
-        public const string ELSE_IF = "elseIf";
         public const string CONSOLE = "print";
```

Comment:
Please, change the name of this class so that it's clear what the constants are

Reply:
done